typewise_alert.test.py

```diff
@@ -1,11 +1,50 @@
 import unittest
 import typewise_alert
 
-
 class TypewiseTest(unittest.TestCase):
-  def test_infers_breach_as_per_limits(self):
-    self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
+  
+  def test_infers_breach_for_TOO_LOW_min_minus_one(self):#(min-1)<(min)
+    self.assertTrue(typewise_alert.infer_breach(19, 20, 30) == 'TOO_LOW')
+    
+  def test_infers_breach_for_TOO_HIGH_max_plus_one(self):#(max)<(max+1)
+    self.assertTrue(typewise_alert.infer_breach(31, 20, 30) == 'TOO_HIGH')
+    
+  def test_infers_breach_for_NORMAL_min_plus_one(self):#(min+1)>(min)
+    self.assertTrue(typewise_alert.infer_breach(21, 20, 30) == 'NORMAL')
+    
+  def test_infers_breach_for_NORMAL_max_minus_one(self):#(max-1)<(max)
+    self.assertTrue(typewise_alert.infer_breach(29, 20, 30) == 'NORMAL')
+
+  def test_check_and_alert_to_console(self):
+    batteryChar1={
+    'coolingType':'PASSIVE_COOLING'
+    }
+    self.assertEqual(typewise_alert.check_and_alert('to_console', batteryChar1, 1),None)
+    
+  def test_check_and_alert_to_controller(self):
+    batteryChar1={
+    'coolingType':'PASSIVE_COOLING'
+    }
+    self.assertEqual(typewise_alert.check_and_alert('to_controller', batteryChar1, 16),None)
```

Comment:
This assert states that check_and_alert converts the values to None.
If you had to state its functionality as data-conversion, what would you state?